packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -102,8 +102,8 @@ export const makeVaultManager = (
   let prioritizedVaults;
   /** @type {MutableQuote=} */
   let outstandingQuote;
-  /** @type {NatValue} */
-  let totalDebt = 0n;
+  /** @type {Amount<NatValue>} */
+  let totalDebt = AmountMath.make(runBrand, 0n);
```

Comment:
`makeEmpty` is preferred, even when we know it's a Nat.

Reply:
I'll get that into https://github.com/Agoric/agoric-sdk/pull/4581 when I rebase it onto master.